pages/apis/rest_api/clusters.md

```diff
@@ -271,25 +270,55 @@ curl "https://api.buildkite.com/v2/organizations/{org.slug}/clusters/{cluster.id
 ```
 
 ```json
-[{
+[
+  {
+    "id": "01885682-55a7-44f5-84f3-0402fb452e66",
+    "graphql_id": "Q2x1c3Rlci0tLTQyZjFhN2RhLTgxMmQtNDQzMC05M2Q4LTFjYzdjMzNhNmJjZg==",
+    "key": "Default Queue",
+    "description": "The default queue for this cluster",
+    "url": "http://api.buildkite.com/v2/organizations/test/clusters/42f1a7da-812d-4430-93d8-1cc7c33a6bcf/queues/01885682-55a7-44f5-84f3-0402fb452e66",
+    "web_url": "http://buildkite.com/organizations/test/clusters/42f1a7da-812d-4430-93d8-1cc7c33a6bcf/queues/01885682-55a7-44f5-84f3-0402fb452e66",
+    "cluster_url": "http://api.buildkite.com/v2/organizations/test/clusters/#{cluster.id}",
+    "dispatch_paused": false,
+    "dispatch_paused_by": null,
+    "dispatch_paused_at": null,
+    "dispatch_paused_note": null,
+    "created_at": "2023-05-03T04:17:55.867Z",
+    "created_by": {
+      "id": "0187dfd4-92cf-4b01-907b-1146c8525dde",
+      "graphql_id": "VXNlci0tLTAxODdkZmQ0LTkyY2YtNGIwMS05MDdiLTExNDZjODUyNWRkZQ==",
+      "name": "Sam Kim",
+      "email": "sam@example.com",
+      "avatar_url": "https://www.gravatar.com/avatar/example",
+      "created_at": "2023-05-03T04:17:43.118Z"
+    }
+  }
+]
+```
+
+Required scope: `read_clusters`
+
+Success response: `200 OK`
+
+### Get a queue
+
+```bash
+curl "https://api.buildkite.com/v2/organizations/{org.slug}/clusters/{cluster.id}/queues/{queue.id}"
+```
+
+```json
+{
   "id": "01885682-55a7-44f5-84f3-0402fb452e66",
   "graphql_id": "Q2x1c3Rlci0tLTQyZjFhN2RhLTgxMmQtNDQzMC05M2Q4LTFjYzdjMzNhNmJjZg==",
   "key": "Default Queue",
   "description": "The default queue for this cluster",
   "url": "http://api.buildkite.com/v2/organizations/test/clusters/42f1a7da-812d-4430-93d8-1cc7c33a6bcf/queues/01885682-55a7-44f5-84f3-0402fb452e66",
   "web_url": "http://buildkite.com/organizations/test/clusters/42f1a7da-812d-4430-93d8-1cc7c33a6bcf/queues/01885682-55a7-44f5-84f3-0402fb452e66",
   "cluster_url": "http://api.buildkite.com/v2/organizations/test/clusters/#{cluster.id}",
-  "dispatch_paused": true,
-  "dispatch_paused_by": {
-    "id": "0187dfd4-92cf-4b01-907b-1146c8525dde",
-    "graphql_id": "VXNlci0tLTAxODdkZmQ0LTkyY2YtNGIwMS05MDdiLTExNDZjODUyNWRkZQ==",
-    "name": "Sam Kim",
-    "email": "sam@example.com",
-    "avatar_url": "https://www.gravatar.com/avatar/example",
-    "created_at": "2023-05-03T04:17:43.118Z"
-  },
-  "dispatch_paused_at": "2023-05-03T04:19:43.118Z",
-  "dispatch_paused_note": "Paused for maintenance",
+  "dispatch_paused": false,
+  "dispatch_paused_by": null,
+  "dispatch_paused_at": null,
+  "dispatch_paused_note": null,
```

Comment:
Is this the only part I should be paying attention to for this PR? Oh, plus the response under "Resume a paused queue."

Reply:
And the `Pause a queue` section is new 😄